css-cascade-7/Overview.bs

```diff
@@ -0,0 +1,402 @@
+spec:selectors-4; type:dfn; text:combinator
+spec:html; type:element; text:style
+spec:css-scoping-1; type:dfn; text:shadow host
+</pre>
+
+<pre class=ignored-specs>
+spec:mediaqueries-5
+spec:css-values-4
+spec:css-fonts-4
+</pre>
+
+<h2 id="intro" oldids='filtering,fragments,stages-examples,actual,used,computed,cascaded,declared,specified,value-stages,all-shorthand,aliasing,shorthand,content-type,import-processing,conditional-import,at-import,defaulting,initial-values,inheriting,defaulting-keywords,initial,inherit,inherit-initial,default'>
+Introduction and Missing Sections</h2>
+
+	Issue: This is a diff spec over <a href="https://www.w3.org/TR/css-cascade-6/">CSS Cascading and Inheritance Level 6</a>.
+	It is currently an Exploratory Working Draft:
+	if you are implementing anything, please use Level 6 as a reference.
+	We will merge the Level 6 text into this draft once it reaches CR.
+
+<!-- Big Text: Stylesheet 
+
+ ███▌  █████▌ █   ▐▌ █▌    █████▌  ███▌  █▌  █▌ █████▌ █████▌ █████▌
+█▌  █▌   █▌   ▐▌  █  █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+█▌       █▌    █ ▐▌  █▌    █▌     █▌     █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌    ▐▌█   █▌    ████    ███▌  █████▌ ████   ████     █▌  
+    █▌   █▌     █▌   █▌    █▌         █▌ █▌  █▌ █▌     █▌       █▌  
+█▌  █▌   █▌     █▌   █▌    █▌     █▌  █▌ █▌  █▌ █▌     █▌       █▌  
+ ███▌    █▌     █▌   █████ █████▌  ███▌  █▌  █▌ █████▌ █████▌   █▌  
+-->
+
+<h2 id='stylesheet-section'>
+Stylesheet</h2>
+
+	A <dfn export>stylesheet</dfn>
+	is a collection of style rules. 
+	
+	These collections may exist as:
+	<ol>
+		<li>An instance of file with the <a href="https://www.w3.org/TR/css-cascade-5/#content-type">CSS Style Sheet Content-Type</a>, embedded within a document.
+		<li>A collection of CSS rules within a document.
+		<li><a href="https://www.w3.org/TR/cssom/#the-stylesheet-interface">CSSOM Stylesheet</a> objects.
+		<li>Any of the above may be subdivided into <dfn>named stylesheet</dfn> with the ''@sheet'' rule.
+	</ol>
+
+	A <dfn>sheet-name</dfn> is a string identifying
+	 inner named sheets within a stylesheet.
+
+	Syntactically, a stylesheet must include an explicit [=sheet-name=], and is represented
+	by the <<sheet-name>> in ''@sheet'' and ''@import'' rules,
+	which is a single <<ident>> token:
+
+	<pre class='prod'>
+		<dfn><<sheet-name>></dfn> = <<ident>>
+	</pre>
+
+<h3 id="at-ruledef-sheet">
+Named Stylesheets: the `@sheet` block at-rule</h3>
+
+	The ''@sheet'' [=block at-rule=]
+	assigns its style rules to a particular [=named stylesheet=].
+	This sheet-assignment syntax is:
+
+	<pre class='prod'>
+		@sheet <<sheet-name>> {
+			<<rule-list>>
+		}
+	</pre>
+
+	Such ''@sheet'' block rules have the same restrictions and processing
+	as a [=conditional group rule=] [[CSS-CONDITIONAL-3]]
+	with a false condition.
+
+Note: This means that rules within an ''@sheet'' block are not applied 
+until their [=named stylesheet=] is explicitly embedded into a document.
```

Comment:
I'm not sure if "embedded into a document" is the right condition here. In the example below, the `@sheet` rule *is* part of the document; its contents are accessible via CSSOM. Maybe "imported into another context" instead?

Reply:
Yeah this one is tricky to phrase. I think "imported into another context" is more correct, so I'm going with that for now, but this might change in the future.

Reply:
Maybe drop `into another context`?
Because you can also import it into the same context, right?

Reply:
Dropped "into another context", as "imported" is sufficient, thanks @romainmenke!